theme-base/components/input/_chips.scss

```diff
@@ -37,6 +37,27 @@
         }
     }
 
+    .p-chips-icon-left > i:first-of-type, 
+    .p-chips-icon-left > svg:first-of-type {
+        left: nth($inputPadding, 2);
+        color: $inputIconColor;
+    }
+
+    .p-chips-icon-right > i:first-of-type, 
+    .p-chips-icon-right > svg:first-of-type {
+        right: nth($inputPadding, 2);
+        color: $inputIconColor;
+    }
+
+    .p-chips-icon-left > .p-chips > .p-inputtext,
+    .p-chips-icon-left > label {
+        padding-left: nth($inputPadding, 2) * 2 + $primeIconFontSize;
+    }
+
+    .p-chips-icon-right > .p-chips > .p-inputtext{
+        padding-right: nth($inputPadding, 2) * 2 + $primeIconFontSize;
+    }
```

Comment:
I think this all has to move "outside" the brackets see right below the `&.p-invalid.p-component` the `&` means add that style on to the one its inside. So I think all these chips styles need to OUTSIDE the main `p-chips` look at inputtext for an example.

Reply:
oh yea your right I was thinking of this wrong. Ill update this soon and figure out how to test it

Reply:
Hopefully my test instructions should be easy to follow!